libs/sdk-ui-filters/src/AttributeFilter/types.ts

```diff
@@ -42,6 +42,15 @@ export interface IAttributeFilterBaseProps
  * @public
  */
 export interface IAttributeFilterCoreProps {
+    /**
+     * @internal
+     *
+     * @remarks
+     * Internal purpose that is used for marking if filter reset children filters after changed
+     * Default value is "true"
+"     */
+    resetChildrenFilters?: boolean;
```

Comment:
I would rather call it `resetOnParentFilterChange` which is probably more appropriate